src/javascript/app/pages/trade/digit_trade.js

```diff
@@ -8,7 +8,7 @@ const localize             = require('../../../_common/localize').localize;
 const DigitDisplay = (() => {
     let $container;
 
-    const initTable = (id_render, calculated_height, poc) => {
+async function initTable(id_render, calculated_height, poc){
```

Comment:
You can keep the arrow syntax as before, just add `async` before the props like:
```
const foo = async (props) => {
  // do something
}
```

Reply:
@akmal-binary Your offer is accepted, thanks